Gems/Atom/Feature/Common/Assets/Shaders/MorphTargets/MorphTargetSRG.azsli

```diff
@@ -33,7 +33,9 @@ struct MorphTargetDelta
     // 8 bit padding plus 8 bits per component for bitangent deltas
     uint m_compressedPadBitangentDeltaXYZ;
     // Extra padding so the struct is 16 byte aligned for structured buffers
-    uint3 m_pad;
+    uint m_pad1;
```

Comment:
A lot of pad floats seem to be broken in this PR. Any reasons why and could we use [[pad_to(16)]] in this case?